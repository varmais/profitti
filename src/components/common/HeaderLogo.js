import React from 'react';
import { Image } from 'react-native';

export default () => (
  <Image source={require('../../assets/images/trol-logo-full.png')} style={{height: 35}} resizeMode="contain" />
);
