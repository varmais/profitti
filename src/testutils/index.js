import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

global.React = React;
global.renderer = renderer;

export function createNavigationProp (params = {}) {
  return {
    state: {params: params},
    navigate: () => {}
  }
}