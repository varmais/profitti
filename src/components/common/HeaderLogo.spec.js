import '../../testutils';
import HeaderLogo from './HeaderLogo';

describe('components:common:HeaderLogo', () => {
  test('renders HeaderLogo', () => {
    const snapshot = renderer.create(<HeaderLogo />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});