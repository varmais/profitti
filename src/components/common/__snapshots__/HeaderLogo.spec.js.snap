// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:common:HeaderLogo renders HeaderLogo 1`] = `
<Image
  resizeMode="contain"
  source={1}
  style={
    Object {
      "height": 35,
    }
  }
/>
`;
